content/intro-to-storybook/vue/fr/conclusion.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
+
+[📕 **Projet GitHub : chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/vue)
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
+Storybook est un puissant outil pour React, React Native, Vue, Angular, Svelte et beaucoup d'autres frameworks. Il y a une communauté de développeurs prospère et une nombre important d'addons. Cette introduction égratigne la surface de ce qui est possible. Nous sommes convaincus qu'une fois que vous aurez adopté Storybook, vous serez impressionnés par la manière de produire des interfaces utilisateurs durables.
+
+## En apprendre plus
+
+Vous voulez en apprendre plus ? Voici des ressources utiles :
+
+- [**Documentation Officielle de Storybook**](https://storybook.js.org/basics/introduction/) possède une documentation de l'API, des liens de la communauté et une galerie d'addons.
```

Comment:
@jeanphibaconnais when you're able, can you please adjust the link for "Documentation Officielle de Storybook" to point at https://storybook.js.org/docs/vue/get-started/introduction?

Reply:
Hi @jonniebigodes , i've just correct the link.

Reply:
Thank you for updating it.